flink-contrib/flink-storm/src/test/java/org/apache/flink/storm/wrappers/WrapperSetupHelperTest.java

```diff
@@ -193,24 +189,22 @@ public void testCreateTopologyContext() {
 			Utils.sleep(++counter * 10000);
 			cluster.shutdown();
 
-			if (TestSink.result.size() == 8) {
+			if (TestSink.result.size() >= 4) {
```

Comment:
The Storm executor sometimes returned more results for me. I've adjusted it to a fixed size again. I think the important thing here is that we check all the returned results.